config/jobs/periodic/housekeeping/cleanup-gcs-logs-periodics.yaml

```diff
@@ -0,0 +1,57 @@
+periodics:
+  - name: periodic-gcs-logs-cleanup
+    cluster: k8s-ppc64le-cluster
```

Comment:
This does not require the ppc64le architecture, wondering whether we can limit this job to run on the IKS cluster

Reply:
Yes it should be fine to run on IKS cluster. I shall verify and amend.

Reply:
Yes it is working fine on IKS cluster.